cmake/RunLint.cmake

```diff
@@ -1,18 +1,27 @@
-get_filename_component(LINT_DIR ${LINT_DIR} ABSOLUTE)
-get_filename_component(LINT_PREFIX ${LINT_DIR} PATH)
+get_filename_component(LINT_DIR "${LINT_DIR}" ABSOLUTE)
+get_filename_component(LINT_PREFIX "${LINT_DIR}" PATH)
```

Comment:
I know that it does, but still prefer the quotes. Why do you think this will be less reliable?